modules/ROOT/pages/alerts-on-runtime-manager.adoc

```diff
@@ -169,15 +169,15 @@ The available alert events vary depending on the *Server Type* field. The alerts
 +
 image::cpu-usage.png[cpu]
 +
-To enable this condition, specify values in the duration and capacity threshold parameters that, when crossed, trigger the alert. 
+To enable this condition, specify values in the duration and capacity threshold parameters that, when crossed, trigger the alert.
 
 
 
 * *Memory Usage* - Total memory usage in MB on the server.
 +
 image::memory-usage.png[memo]
 +
-To enable this condition, specify values in the duration and capacity threshold parameters that, when crossed, trigger the alert. 
+To enable this condition, specify values in the duration and capacity threshold parameters that, when crossed, trigger the alert.
```

Comment:
This is the same text that's on line 172. There is probably a reason for it. Just checking.

Reply:
Seems applicable to both.